src/packages/cli/src/__tests__/__helpers__/integrationTest.ts

```diff
@@ -323,15 +335,19 @@ async function setupScenario(kind: string, input: Input, scenario: Scenario) {
             ? input.database.datasource.url(ctx)
             : input.database.datasource.url,
         )
-
   const schemaBase = `
-        generator client {
-          provider = "prisma-client-js"
-          output   = "${ctx.fs.path()}"
-        }
+    generator client {
+      provider = "prisma-client-js"
+      output   = "${ctx.fs.path()}"
+      ${
+        input.database.name === 'sqlserver'
+          ? `previewFeatures = ["microsoftSqlServer"]`
+          : ''
+      }
```

Comment:
Same here. This will ideally go inside MSSQL tests rather than being in the common utility.